### ISSUE TITLE
Template for relevant papers?

### ISSUE THREAD

**Issue by robcrystalornelas** [SORTEE-Github-Hackathon/manuscript#34]
Is it possible for us to set up a template so that every time we create a new discussion thread for `relevant papers`, the boilerplate language for describing the paper (pasted below) is automatically copied over?

---

### Title: Include the manuscript's title

Study Link: Include the https:// link that brings us to the page of the manuscript
Citation: `@doi:replace`

> Citation must follow the [Manubot-style citation](https://github.com/SORTEE-Github-Hackathon/manuscript/blob/main/USAGE.md#citations). Leave it in blank, if unsure.

Suggested keywords that help identify the relevance of this paper to Ecology and Evolution:
* Keyword 1 (replace me, copy and paste more than three if needed)
* Keyword 2 (replace me, copy and paste more than three if needed)
* Keyword 3 (replace me, copy and paste more than three if needed)

### Which areas of expertise are particularly relevant to the paper?

- [x] ecology and/or evolution;
- [ ] biostatistics;
- [ ] informatics and computational research;
- [ ] open science and reproducibility;
- [ ] other: replace_with_your_suggestion.

# Summary

### Suggested questions to answer about each paper:
- What is the general main finding or takeaway of the paper?
- What did they analyse and how did they do it?
- What does this paper suggest to improve the issue with reproducibility in science?
- Do you have any concerns about methodology or the interpretation of these results beyond this analysis?

### Any comments or notes?

**Comment by pedrohbraga** [comment]
Unfortunately, no, @robcrystalornelas. 

I had looked for this when setting up the repository and I have found it is not possible.

This is a [feature request](https://github.com/github/feedback/discussions/2838) that exists for Github Discussions. Early this year, they said that the implementation was on their roadmap, so hopefully, it becomes available in the near future.

In the meanwhile, we may have to rely on the recommendations in thread #14 when suggesting new papers, unless an alternative is implemented.

**Comment by robcrystalornelas** [comment]
Great, thanks for that link Pedro! I'll for sure keep referring to thread #14 in the meantime--cool to know they are thinking about implementing it next year. 👍🏻

**Comment by Aariq** [comment]
Yeah, you can make templates for different kinds of issues, but I guess not yet for Discussions.
